library/std/src/sys/xous/os.rs

```diff
@@ -20,7 +42,14 @@ mod c_compat {
     }
 
     #[no_mangle]
-    pub extern "C" fn _start() {
+    pub extern "C" fn _start(eh_frame: usize) {
+        #[cfg(feature = "panic_unwind")]
+        unsafe {
+            super::eh_unwinding::EH_FRAME_SETTINGS.init(eh_frame);
+            println!("Setting custom eh frame finder");
+            unwind::set_custom_eh_frame_finder(&super::eh_unwinding::EH_FRAME_SETTINGS).ok();
+        }
+        crate::println!("About to enter main()");
```

Comment:
I suppose these 2 `println!` lines were for your own debugging?

Reply:
You're very right. Thank you for pointing that out. I've removed those.